build/dockerfiles/KServe/README.md

```diff
@@ -0,0 +1,56 @@
+# Kitops ClusterStorageContainer image for KServe
+
+The Dockerfile in this directory is used to build an image that can run as a ClusterStorageContainer for [KServe](https://kserve.github.io/website/master/).
+
+## Building
+To build the image, `docker` or `podman` is required. From the root of this repository, run
+```bash
+podman build . \
+  -f build/dockerfiles/KServe/kserve.Dockerfile \
+  -t $KIT_KSERVE_IMAGE \
```

Comment:
We should mention that this value should be set or substituted

Reply:
Done; ideally this step isn't important since we'd eventually be building this image and publishing it on ghcr.io, which would move build to the bottom of this doc.